src/main/java/ithillel/lesson5/ArrayHillelList.java

```diff
@@ -0,0 +1,55 @@
+package ithillel.lesson5;
+
+public class ArrayHillelList implements HillelList{
+
+    private String[] ArrayList = {};
+
+    public ArrayHillelList(){}
+    public ArrayHillelList(String[] items){ArrayList = items.clone();}
+
+    public String[] getAll() {
+        return ArrayList;
+    }
+
+    public int size() {
+        return ArrayList.length;
+    }
+
+    public void add(String item){
+        String[] temp = new String[size() + 1];
```

Comment:
```suggestion
        String[] temp = new String[ArrayList.length + 1];
```
можна просто напряму